lsp/vscode-extension/language-configuration.json

```diff
@@ -1,32 +1,26 @@
 {
   "comments": {
-    // symbol used for single line comment. Remove this entry if your language does not support line comments
     "lineComment": "#"
   },
-  // symbols used as brackets
   "brackets": [
     ["{", "}"],
     ["%{", "}"],
     ["[", "]"],
     ["[|", "|]"],
     ["(", ")"]
   ],
-  // symbols that are auto closed when typing
   "autoClosingPairs": [
     ["{", "}"],
     ["[", "]"],
     ["[|", "|]"],
     ["(", ")"],
-    ["\"", "\""],
-    ["'", "'"]
```

Comment:
Single quotes are used for enum tags in Nickel, like quoting in Lisp, that is `some_function 'Http 'Json`, but isn't a pair opening/closing delimiter.

Reply:
Hm, should we make it some form of keyword/operator then?

Reply:
You're right, but it should rather be an separate rule, I guess. In the vim extension, if you type `'foo`, the whole word + starting quote is getting colored in a special way, which is probably the desired behavior. Let met try to add that.